woof-code/rootfs-skeleton/usr/sbin/remasterpup2

```diff
@@ -525,11 +588,12 @@ $(gettext 'Please wait...')"
 	#modify /root/.packages/ files, as all installed packages are now part of the new cd...
 	cp -a -f /root/.packages /tmp/root/
 	touch /root/.packages/user-installed-packages
+	
 	# 141008: move *.files to ~/.packages/builtin_files/
 	cat /root/.packages/user-installed-packages | \
 	while read -r ONEPKG
 	do
-		ONEFILE="/tmp/root/.packages/`echo "$ONEPKG" | cut -f1 -d '|'`.files"
+		ONEFILE="/tmp/root/.packages/package-files/`echo "$ONEPKG" | cut -f1 -d '|'`.files"
```

Comment:
I think this is easier to read if we use the 
```shell
ONEFILE="/tmp/root/.packages/package-files/$(echo "$ONEPKG" | cut -f1 -d '|').files"
```
syntax rather than
```shell
ONEFILE="/tmp/root/.packages/package-files/`echo "$ONEPKG" | cut -f1 -d '|'`.files"
```
syntax.

Of course this is just an opinion. :smile_cat: